src/main/java/com/wolox/training/models/Users.java

```diff
@@ -0,0 +1,95 @@
+package com.wolox.training.models;
+
+import com.sun.istack.NotNull;
+import com.wolox.training.exceptions.BookAlreadyOwnedException;
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.JoinTable;
+import javax.persistence.ManyToMany;
+
+@Entity
+public class Users {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private long id;
+
+    @Column(nullable = false)
+    @NotNull
+    private String userName;
+
+    @Column(nullable = false)
+    @NotNull
+    private String name;
+
+    @Column(nullable = false)
+    @NotNull
+    private LocalDate birthDate;
+
+    @Column(nullable = false)
+    @NotNull
+    @ManyToMany(cascade = CascadeType.ALL)
+    @JoinTable(name = "book_users",
```

Comment:
Unless you specifically want to rename join table columns or table name, you can skip defining the @JoinTable annotation given everything will be sorted out automatically because of the @manytomany relationship.